src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md

```diff
@@ -54,6 +54,50 @@ The following example shows a PHTML file using the script:
 </script>
 ```
 
+The following example shows a PHTML file using the script with some data:
+
+```html
+<script>
+    require([
+        'jquery',
+        'rowBuilder'
+    ], function ($) {
+        'use strict';
+
+        var data = {
+            formData: [
+                ["field1-name1", "field2-name1", "field3-name1"],
+                ["field1-name2", "field2-name2", "field3-name2"],
+                ["field1-name3", "field2-name3", "field3-name3"],
+                ["field1-name4", "field2-name4", "field3-name4"],
+            ],
+            templateFields: [
+                "field1-name", "field2-name", "field3-name"
+            ]
+        };
+
+        $(".row-builder-package-manager").rowBuilder({
+            "rowTemplate": "#row-template",
+            "rowContainer": "#row-container",
+            "rowParentElem": "<div></div>",
+            "remEventSelector": "a",
+            "btnRemoveSelector": ".action.remove",
+            "formDataPost": data
+        });
+    });
+</script>
+```
+
+This is provided during widget instantiation. For example:
+
+```
```

Comment:
@DashukVolodymyr , could you please add `text` after ```

Reply:
Hi @atwixfirster,  @BarnyShergold 
Thanks for your notes.

I think for this reason the linting tests did not pass!
I added `text` after ``` as requested.

Please proceed.
Thank you!

Reply:
Looks ok to me. ✅ 

Thank you!